blockdev/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
```

Comment:
This looks like a merge conflict artifact. I think we can just drop it - the path is now in `crates/blockdev/Cargo.toml`.